### ISSUE TITLE
Handle ignored headers separately for request and response headers

### ISSUE THREAD

**Issue by woylie** [rhazdon/blue_bird#11]
Currently, it is only possible to ignore a header for both the request and the response. There might be cases where we want to ignore a header in the request, but still display it in the response and vice versa.

So instead of `ignore_headers: ["not-wanted"]`, we need something like

```
ignore_request_headers: ["not-wanted"],
ignore_response_headers: ["not-wanted-either"]
```

or

```
ignore_headers: [
    request: ["not-wanted],
    response: ["not-wanted-either"]
]
```

Which one? Or something else?

**Comment by woylie** [comment]
We could allow two ways of defining the `ignore_headers` option.

If you want to ignore headers for both requests and responses:

```
ignore_headers: ["not-wanted", "not-wanted-either"]
```

If you want to differentiate between request and response headers:

```
ignore_headers: %{
    request: ["not-wanted],
    response: ["not-wanted-either"]
}
```

**Comment by woylie** [comment]
addressed in pr #33
